### PR TITLE
[3709] DQT TRN request jobs & services

### PR DESCRIPTION
**Pull request by mooktakim**
### Context

https://trello.com/c/5peN6dPU/3709-l-dqt-trn-request-jobs-services

### Changes proposed in this pull request

* Jobs and services added to register and retrieve TRN from DQT

### Guidance to review

* Run on console? 🤷‍♀️

### Important business

* Does this PR introduce any PII fields that need to be overwritten or deleted in db/scripts/sanitise.sql?
* Does this PR change the database schema? If so, have you updated the config/analytics.yml file and considered whether you need to send 'import_entity' events?

### CODE REVIEW

**Comment by mooktakim** [comment]
[[L] DQT TRN request jobs & services](https://trello.com/c/5peN6dPU/3709-l-dqt-trn-request-jobs-services)

File: db/schema.rb [hunk 1]
```diff
@@ -210,6 +210,17 @@
     t.index ["name"], name: "index_disabilities_on_name", unique: true
   end
 
+  create_table "dqt_trn_requests", force: :cascade do |t|
```
Comment (gpeng) [inline db/schema.rb:213]: Need to add this table and fields to analytics.yml. Probably without the response but we should check that with @AndrewPetrie1

**Review by gpeng — COMMENTED** [review]
Looks good. Just need to add update analytics.yml too.

File: db/schema.rb (hunk 1 above)
Comment (gpeng) [reply db/schema.rb:213]: We might want to add the response to sanitise.yml too if we're going to consider it PII.

**Comment by mooktakim** [comment]
[Dqt::TrnRequest](https://trello.com/c/C77GxGEK/3706-dqttrnrequest)

**Review by gpeng — APPROVED** [review]

**Pull request merged**